src/main/java/fr/techad/edc/popover/utils/OpenUrlAction.java

```diff
@@ -8,27 +8,103 @@
 
 import javax.inject.Inject;
 import java.awt.*;
-import java.io.IOException;
+import java.io.*;
+import java.net.HttpURLConnection;
 import java.net.URISyntaxException;
 import java.net.URL;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.Executors;
 
 /**
  * TECH ADVANTAGE
  * All right reserved
  * <p>
  * Utility class to open an url in a browser.
  */
-public class OpenUrlAction {
+public class OpenUrlAction{
     private static final Logger LOGGER = LoggerFactory.getLogger(OpenUrlAction.class);
 
     private final Browser browser;
     private final HelpConfiguration helpConfiguration;
+    private Process edcDesktopProcess = null;
 
     @Inject
     public OpenUrlAction(Browser browser, HelpConfiguration helpConfiguration) {
         super();
         this.browser = browser;
         this.helpConfiguration = helpConfiguration;
+
+    }
+
+    private HttpURLConnection postViewerURL(String url) throws IOException {
+        String viewerUrl = "http://localhost:60000/viewerurl";
```

Comment:
and if the viewser works on different port, you have to enable the configuration of this url.